src/client/Microsoft.Identity.Client/TelemetryCore/TelemetryClient/TelemetryDatapoints.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.IdentityModel.Abstractions;
+
+namespace Microsoft.Identity.Client.TelemetryCore.TelemetryClient
+{
+    /// <summary>
+    /// Stores the cache details to log to <see cref="ITelemetryClient"/>.
+    /// </summary>
+    public class TelemetryDatapoints
```

Comment:
How about `CacheTelemetryDatapoints` ?

Reply:
Will this ever store more stuff? e.g. aggregate errors or track latency of other sub operations? If no, agree with above comments to make type more descriptive. If yes, class header should be updated to reflect a broader scope than cache.

Reply:
We can keep this as TelemetryDatapoints as we can use this class to capture more information in future.